sdk/managed/test/Microsoft.WindowsAzure.Mobile.Net45.Test/App.xaml.cs

```diff
@@ -24,7 +24,7 @@ public partial class App : Application
         static App()
         {
             Harness = new TestHarness();
-            Harness.Platform = string.Format("Net 45|sdk v{0}|", TestPlatform.GetMobileServicesSdkVersion(typeof(App).GetTypeInfo().Assembly));
+            Harness.Platform = TestPlatform.Net45;

```

Comment:
SDK version information is temporarily removed from the reports, will be added in the future.